crt_portal/cts_forms/mail.py

```diff
@@ -74,20 +74,32 @@ def crt_send_mail(report, template, purpose=TMSEmail.MANUAL_EMAIL):
         # replace newlines, \n, with <br> so the API will generate formatted emails
         html_message = message.replace('\n', '<br>')
 
+    if settings.EMAIL_BACKEND != 'tms.backend.TMSEmailBackend':
+        TMSEmail.create_fake(subject=subject,
+                             body=message,
+                             html_body=html_message,
+                             report=report,
+                             purpose=purpose
+                             ).save()
+        return [1]  # This pretends we've sent one email to one recipient.
+
+    if dry_run:
+        return [0]  # This pretends we've sent one email to one recipient.
```

Comment:
this would mean we didn't send an email right?

Reply:
correct - changed this to be a proper docstring comment, thanks!